src/Raven.Studio/typescript/models/database/tasks/periodicBackup/backupConfiguration.ts

```diff
@@ -30,6 +30,8 @@ abstract class backupConfiguration {
     snapshot = ko.observable<snapshot>();
     
     mentorNode = ko.observable<string>();
+    //pinMentorNode = ko.observable<boolean>(); // todo new..??
```

Comment:
I assume we track that part in different issue, right?

Reply:
can we uncomment that?

Reply:
fixed